recipes/primesieve/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc_static_runtime, is_msvc
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class PackageConan(ConanFile):
+    name = "primesieve"
+    description = "Fast prime number generator"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kimwalisch/primesieve"
+    topics = ("math", "prime-numbers", "sieve-of-eratosthenes")
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Missing package_type
```suggestion
    package_type = "library"
    settings = "os", "arch", "compiler", "build_type"
```

Reply:
Done